nitrous-post-create.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Ideally yeah as this is what installs Brunch onto the Nitrous dev environment that's being created for the user.